app/admin_summary.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python2.7
```

Comment:
It may be better to name this "statistics" instead of "summary", for both the file name and the URL. "Summary" is a bit vague.